libraries/apollo-runtime-java/src/main/java/com/apollographql/apollo3/runtime/java/interceptor/ApolloInterceptor.java

```diff
@@ -1,24 +1,23 @@
 package com.apollographql.apollo3.runtime.java.interceptor;
 
 import com.apollographql.apollo3.api.ApolloRequest;
+import com.apollographql.apollo3.api.Operation;
 import com.apollographql.apollo3.runtime.java.ApolloCallback;
 import org.jetbrains.annotations.NotNull;
 
-import java.util.concurrent.Executor;
-
 /**
- * ApolloInterceptor is responsible for observing and modifying the requests going out and the corresponding responses
- * coming back in. Typical responsibilities include adding or removing headers from the request or response objects,
- * transforming the returned responses from one type to another, etc.
+ * ApolloInterceptor is responsible for observing and modifying the requests going out and the corresponding responses coming back in.
```

Comment:
Woops it looks like we have different formatter settings :(.  Or maybe we should disable reformatting of Javadocs?

Reply:
Not sure I did reformat that. I copy/pasted it from v2 so hopefully it didn't go through the current reformating rules and everything is ok 👍